algorithms/linfa-ftrl/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+//! # Follow the regularized leader - proximal
+//!
+//! ## The Big Picture
+//!
+//! `linfa-ftrl` is a crate in the [`linfa`](https://crates.io/crates/linfa) ecosystem, an effort to create a toolkit for classical Machine Learning implemented in pure Rust, akin to Python's `scikit-learn`.
+//!
+//! ## Current state
+//! `linfa-ftrl` provides a pure Rust implementation of an [algorithm](struct.FollowTheRegularizedLeader.html).
+//!
+//! ## Examples
+//!
+//! There is an usage example in the `examples/` directory. To run, use:
+//!
+//! ```bash
+//! $ cargo run --example winequality
+//! ```
+//!
```

Comment:
Try replacing with `#![doc = include_str!("../README.md")]`. This syncs the README docs with the crate docs and runs the README example as a doctest.